Add changes to conclusion (#37)
11michalis11/ED-EMS-game-paper: sections/conclusion/main.tex

\section{Conclusion}

> The motivation behind this study has been that emergency departments are
> under a lot of pressure to satisfy some regulations.
> This paper shows how this can negatively impact the pathway of both the

Reviewer: ```suggestion
The motivation behind this study has been that emergency departments are
under a lot of pressure to treat patients. This is in practice often centrally controlled through a mechanism of some sort of performance measure target.
This paper shows how this can negatively impact the pathway of both the
```

ambulance patients and the ambulance service itself.
Due to some managerial decision making that takes place at the ED, ambulances
stay blocked outside of the ED at the hospital's parking zone in an attempt
to satisfy these regulations.
The main contributions of this paper are:
\begin{itemize}

>     \item A queueing model with 2 consecutive waiting spaces where one would
>     serve as a parking space for the ambulances
>     \item Analytic performance measure formulas for the queueing model
>     \item A 3-player game theoretic model between the EMS and two EDs
>     \item Numerical experiments showing emergent behaviour of gaming between

Reviewer: ```suggestion
    \item A queueing model with 2 consecutive waiting spaces where one would
    serve as a parking space for the ambulances;
    \item Analytic performance measure formulas for the queueing model;
    \item A 3-player game theoretic model between the EMS and two EDs;
    \item Numerical experiments showing emergent behaviour of gaming between
```

\ref{sec:queueing_model}).
Furthermore, the formulas for the performance measures of the queueing model
are also derived (sections \ref{sec:waiting_time}, \ref{sec:blocking_time},
\ref{sec:proportion_within_target}).
This novel queuing model is the first contribution of the paper.
The game theoretic model is then applied to a healthcare scenario by looking at
the interface between the EDs and the EMS.
The inefficiencies that emerge from the perspective of the EMS were explored
along with ways to apply some incentive mechanisms to improve them.
The key findings from this paper that were observed when playing the game
between two EDs and the EMS are:
\begin{itemize}
    \item Inefficiencies can be learned and emerge naturally
    \item Targeted incentivisation of behaviours can help escape inefficiencies
\end{itemize}
The former relates to the results of asymmetric replicator dynamics that showed
that inefficient scenarios can arise by playing the game while the latter
implies that the learned inefficiencies can be escaped by carefully applying
certain incentives to the players.
This applied game theoretic model is the second main contribution of this paper.

> The model presented here assumes the presence of two players that can receive

Reviewer: ```suggestion
The model presented here assumes the presence of just two players that can receive
```

Reviewer 2: Alex suggests that `just` may be insensitive here. Will make it `only two players`
